strings.txt

```diff
@@ -25859,3 +25859,38 @@
     el = Σχεδιάστε μια διαδρομή με ενδιάμεσο προορισμό.
     sk = Naplánujte si trasu s zastávkou.
     sw = Panga njia ya kufuata yenye eneo la kati.
+
+[subtitle_rent]
+    comment = Do not translate
+    en = Rent
+    ru = Аренда
+
+[rub_month]
+    comment = Do not translate
+    en = ₽/month
+    ru = ₽/мес
+
+[room]
+    comment = Do not translate
+    en = %s room(s) apart.
```

Comment:
Also, it's better to make this in line with the Russia version: "%s-room apartment". This way we get rid of the ugly "(s)".